server/src/main/kotlin/net/horizonsend/ion/server/misc/Audience.kt

```diff
@@ -1,19 +1,24 @@
-package net.horizonsend.ion.server.extensions
+package net.horizonsend.ion.server.misc.extensions
 
 import net.horizonsend.ion.server.IonServer.Companion.Ion
 import net.kyori.adventure.audience.Audience
-import net.kyori.adventure.text.Component.text
 import net.kyori.adventure.text.format.TextColor.color
+import net.kyori.adventure.text.minimessage.MiniMessage
 import org.bukkit.entity.Player
 
 /** Returns the prefix that should be used when displaying server messages sent to this audience in the console. */
-private val Audience.prefix get() = when (this) {
-	is Player -> "to $name: "
-	else -> ""
-}
+private val Audience.prefix
+	get() = when (this) {
+		is Player -> "to $name: "
+		else -> ""
+	}
 
 /** Sends a message to the audience with the given colour. */
-private fun Audience.sendColouredMessage(message: String, color: Int) = sendMessage(text(message, color(color)))
+private fun Audience.sendColouredMessage(message: String, color: Int) = sendMessage(
```

Comment:
No

Reply:
bro this is spotless, if you this fixed, fix spotless

Reply:
*sorry, meant the lines below, you changed it to use minimessage*

Reply:
why shouldnt it use minimessage? Why are we switching all the sudden